app/models/contact.rb

```diff
@@ -11,6 +11,24 @@ class Contact < ActiveRecord::Base
   before_validation :initialize_contact
   validates :applicant, :person, presence: true
   validates_associated :person
+  
+  CONTACT_TYPES = [
+    {
+      name: "Emergency Contact",
+      value: "emergency_contact"
+    },
+    {
+      name: "Reference",
+      value: "reference"
+    },
+    {
+      name: "General Contact",
+      value: "general_contact"
+    }
+  ]
+  #enum contact_type: CONTACT_TYPES.map {|ct| ct[:value]}
```

Comment:
Looks like you left some commented-out code in this pull request.  Please take that out.